.github/workflows/ci.yml

```
@@ -86,18 +86,22 @@ jobs:
             arch: x64
             trixi_test: threaded_legacy
           - version: '1.9'
+            os: ubuntu-latest
+            arch: x64
+            trixi_test: threaded_legacy
+          - version: '1.10'
             os: macOS-latest
             arch: x64
```

```suggestion
            os: macos-latest
            arch: aarch64
```
`macos-latest` nowadays refers to Apple Silicon machines, see also https://github.com/actions/runner-images?tab=readme-ov-file#available-images